docs/getting-started.mdx

```diff
@@ -1,81 +1,85 @@
 # Getting Started
```

Comment:
Improving this getting started file helped me identify with our users and helped me to understand the challenges they face.